firebase-chat/firebase-chat/Model Controllers/NetworkController.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  NetworkController.swift
+//  firebase-chat
+//
+//  Created by Joe on 4/25/20.
+//  Copyright © 2020 AlphaGradeINC. All rights reserved.
+//
+
+import Foundation
+
+class NetworkController {
+    
+     static let baseURL = URL(string: "https://fir-chat-bbed5.firebaseio.com/")!
+    
+    var messageThreads: [MessageThread] = []
+
+    func fetchMessageThreads(completion: @escaping () -> Void) {
```

Comment:
Fix indentation.... Tip: Select whole code file with Command-A then re-indent with Control-i
Also generally you would usually want at least a boolean flag in your completion handler, to indicate whether or not the fetch was a success or not.